spec/tictactoe_spec.rb

```diff
@@ -43,5 +43,34 @@
     end
   end
 
+  describe "#check_for_winner" do
+    it "return winner symbol if there is a winning combo on board" do
+       game = Tic_Tac_Toe.new
 
+       # checks horizontal win
+       winning_board = ["X", "X", "X", " ", " ", " ", " ", " ", " "]
+       #checks diagonal win
+       winning_board2 = [" ", " ", "X", " ", "X", " ", "X", " ", " "]
+       #checks vertical column 
+       winning_board3 = ["O", " ", " ", "O", " ", " ", "O", " ", " "]
+
+       expect(game.check_for_winner?(winning_board)).to eq("X")
+       expect(game.check_for_winner?(winning_board2)).to eq("X")
+       expect(game.check_for_winner?(winning_board3)).to eq("O")
+    end
+    
+    it "return false if there is no winning combo on board" do
+       game = Tic_Tac_Toe.new
+
+       no_winning_board = ["X", "X", " ", " ", " ", " ", " ", " ", " "]
+       no_winning_board2 = ["O", "X", "O", "O", "X", "O", "X", "O", "X"]
+
+       expect(game.check_for_winner?(no_winning_board)).to eq(false)
+       expect(game.check_for_winner?(no_winning_board2)).to eq(false)
```

Comment:
```suggestion
       expect(game.check_for_winner?(no_winning_board)).to be_falsey
       expect(game.check_for_winner?(no_winning_board2)).to be_falsey
```

Reply:
Saw this right after I committed. Will change right meow and re-commit